projects/objects/animals/protos/Horse.proto

```diff
@@ -0,0 +1,132 @@
+#VRML_SIM R2022b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# A brown horse.
+# This object is physics-enabled so it can be pushed.
+# template language: javascript
+
+PROTO Horse [
+  field SFVec3f    translation 0 0 0
+  field SFRotation rotation    0 0 1 0
+  field SFFloat    scale       1      # Defines the global scaling factor.
+  field SFString   name        "horse"
+  field SFFloat    mass        750    # Defines the mass of the cow in kg.
+]
+{
+  %<
+    let scale = fields.scale.value;
+    if (scale <= 0) {
+      scale = fields.scale.defaultValue;
+      console.error('\'scale\' must be strictly positive. Value reset to ' + scale + '.');
+    }
+  >%
+  Solid {
+    translation IS translation
+    rotation IS rotation
+    name IS name
+    model "horse"
+    scale %<= scale >% %<= scale >% %<= scale >%
+    children [
+      Shape {
+        appearance PBRAppearance {
+          metalness 0
+          roughness 0.7
+          baseColorMap ImageTexture {
+            url "webots://projects/objects/animals/protos/horse/textures/BaseColorHorse.jpg"
+          }
+        }
+        geometry Mesh {
+          url "webots://projects/objects/animals/protos/horse/meshes/Horse.obj"
```

Comment:
all other object files you included are lowercase, for consistency this should be as well